### ISSUE TITLE
max height for non-blocks code snippets

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt#6010]
some of the snippets that are particularly long make the page hard to follow (especially in arcade for snippets with images); it might be nice to had a ``max-height: 40em`` or something of the sort to the text code snippets so they are available without making the page > 2x as long per expanded snippet.

![2019-10-04 11 18 27](https://user-images.githubusercontent.com/5615930/66230363-e00ee200-e698-11e9-8dfe-a79f62f8caec.gif)

![2019-10-04 11 18 52](https://user-images.githubusercontent.com/5615930/66230377-e3a26900-e698-11e9-8ce9-802e7d8692d0.gif)

Maybe also for block snippets that are super large, but that might be more debatable?

![2019-10-04 11 24 51](https://user-images.githubusercontent.com/5615930/66230747-ae4a4b00-e699-11e9-8095-ce298a23f3a9.gif)

**Comment by pelikhan** [comment]
Sounds reasonable; however they need to be full expanded in print mode.
